printer.py

```diff
@@ -59,6 +60,11 @@ def move(direction):
 def status(size):
     print("Your katamari is {0}cm".format(str(size)))
 
+def bump(item):
+    status=["bumps and","rubs and","wiggles then","stalls then"]
+    verb = status[random.randrange(len(status))]
```

Comment:
I think it should be random.choice(status) rather than status[random.randrange(len(status))]